test.md

```diff
@@ -0,0 +1 @@
+this file should be ignored by git
```

Comment:
- [ ] It is also required that this file should be ignored. Meaning it should not reflect on your repository. The way to do this is by adding the file to the `.gitignore`  you created. Do not also forget to add node_modules folder to the `.gitignore` file.

Reply:
@Qoosim I don't fully understand that. please a little elaboration would help especially node_modules folder part